test/stores/widgets/StopGapWidgetDriver-test.ts

```diff
@@ -388,6 +391,94 @@ describe("StopGapWidgetDriver", () => {
         });
     });
 
+    describe("sendDelayedEvent", () => {
+        let driver: WidgetDriver;
+        const roomId = "!this-room-id";
+
+        beforeEach(() => {
+            driver = mkDefaultDriver();
+            jest.spyOn(SdkContextClass.instance.roomViewStore, "getRoomId").mockReturnValue(roomId);
```

Comment:
Please remember to always called `restoreAllMocks()` in an `afterEach()` if you mock or spy on anything, otherwise your mocks will bleed into other tests.

Reply:
It turns out I don't need this after all, so I'll just get rid of it.